examples/causal_inference/interventional_distribution.ipynb

```diff
@@ -0,0 +1,1561 @@
+{
```

Comment:
<p>I wonder if this introduction kind of buries the lede a bit. It's the introduction of the famous 'do'-operator. Makes PyMC a direct player in the Causal inference space. Feels like the intro should be punchier. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Just for context, we have a much punchier announcement of the do operator that will appear as a PyMC Labs blogpost. So this example is less of an announcement than a resource which will hopefully be useful for some time to come. That said, I agree that it's currently a bit meek. I'll up the level of excitement.</p><p> </p>